test/e2e/main_test.go

```diff
@@ -80,3 +83,13 @@ func TestMain(m *testing.M) {
 	exitCode = m.Run()
 
 }
+
+// utility functions
+
+func getAllocation(f *stablev1alpha1.Fleet) *allocationv1alpha1.GameServerAllocation {
```

Comment:
should it be part of framework.go?

Reply:
That is a much better spot for it! Moving.